api-client/src/models/TunewsModel.ts

```diff
@@ -17,11 +18,15 @@ class TunewsModel {
     eNewsNo: string
   }) {
     const { id, date, title, tags, content, eNewsNo } = params
+
+    const parsedTitle = parseHTML(title)
+    const parsedContent = parseHTML(content)
```

Comment:
I think this should happen in the corresponding Endpoint classes which create TunewsModel. It is not expected that a constructor is doing heavy work or even throw errors if the html is invalid.

Also this could influence tests which use this model and do not expect that this happends.